administrator/components/com_csp/src/Helper/ReporterHelper.php

```diff
@@ -126,7 +126,7 @@ public static function getCspUnsafeEvalStatus()
 			->select('COUNT(*)')
 			->from($db->quoteName('#__csp'))
 			->where($db->quoteName('blocked_uri') . ' = ' . $db->quote("'unsafe-eval'"))
-			->where($db->quoteName('published') . ' = ' . $db->quote('1'));
+			->where($db->quoteName('published') . ' = 1'));
```

Comment:
```suggestion
			->where($db->quoteName('published') . ' = 1');
```
Fix PHP syntax error.